src/ibc_api/metadata.py

```diff
@@ -0,0 +1,90 @@
+"""Functions to fetch metadata about the available IBC datasets."""
+
+import json
+import os
+
+REMOTE_ROOT = (
+    "https://api.github.com/repos/individual-brain-charting/docs/contents"
+)
+
+LOCAL_ROOT = os.path.join(os.path.dirname(__file__), "data")
+os.makedirs(LOCAL_ROOT, exist_ok=True)
+
+
+def _load_json(data_file):
+    """Read a given json file"""
+    with open(data_file, "r") as f:
+        data = json.load(f)
+
+    return data
+
+
+def ibc_subjects():
```

Comment:
Why do we need a function to define a constant ?

Reply:
Ok yes we don't! Will fix that. Thanks!